layouts/partials/docs/right-nav.html

```diff
@@ -6,14 +6,14 @@
         {{ $isPythonPackage := hasPrefix .Path "docs/reference/pkg/python/pulumi" }}
         {{ if not (or $isCLICommand $isNodePackage $isPythonPackage $.Page.Params.no_edit_this_page) }}
             <li>
-                <a class="text-gray-600 hover:text-blue-700 text-xs" href="{{ $.Site.Params.repositoryURL }}/edit/{{ $.Site.Params.repositoryBranch }}/content/{{ .Path }}" target="_blank">
+                <a data-track="edit-page" class="text-gray-600 hover:text-blue-700 text-xs" href="{{ $.Site.Params.repositoryURL }}/edit/{{ $.Site.Params.repositoryBranch }}/content/{{ .Path }}" target="_blank">
```

Comment:
I'd like to know more about why we care about things like this, along with some others in this PR. What questions are we asking by tracking clicks to Edit Page, Request a Change, etc.?